internal/handlers/user.go

```diff
@@ -267,45 +267,24 @@ func (c User) GetUsersList(repository domain.UserRepository) users.GetAllUsersHa
 	}
 }
 
-func (c User) DeleteUserByID(repo domain.UserRepository) users.DeleteUserHandlerFunc {
+func (c User) DeleteUserByID(repository domain.UserRepository) users.DeleteUserHandlerFunc {
 	return func(p users.DeleteUserParams, access interface{}) middleware.Responder {
-		isAdmin, err := authentication.IsAdmin(access)
+		ctx := p.HTTPRequest.Context()
+		userId, err := authentication.GetUserId(access)
 		if err != nil {
-			c.logger.Error("error while getting authorization", zap.Error(err))
-			return users.NewDeleteUserDefault(http.StatusInternalServerError).
-				WithPayload(&models.Error{Data: &models.ErrorData{Message: "Can't get authorization"}})
-		}
-		if !isAdmin {
-			c.logger.Error("user is not admin", zap.Any("access", access))
-			return users.NewDeleteUserDefault(http.StatusForbidden).
-				WithPayload(&models.Error{Data: &models.ErrorData{Message: "You don't have rights"}})
+			c.logger.Error("get user id error during deleting user account", zap.Error(err))
```

Comment:
Could you please remove the word 'account' from the codebase for consistency. Thanks.